src/plots/plots.js

```diff
@@ -743,7 +773,7 @@ plots.supplyLayoutGlobalDefaults = function(layoutIn, layoutOut) {
     coerce('margin.autoexpand');
 
     // called in plotAutoSize otherwise
-    if(autosize !== 'initial') plots.sanitizeMargins(layoutOut);
+    if(layoutOut.width && layoutOut.height) plots.sanitizeMargins(layoutOut);
```

Comment:
Is this still necessary?

Looks like `layoutOut.width` and `layoutOut.height` will now always be defined at this stage.